src/aind_data_transfer_service/configs/core.py

```diff
@@ -0,0 +1,258 @@
+"""Core models for using V2 of aind-data-transfer-service"""
```

Comment:
The stuff in the configs folder are mostly legacy. Maybe we can create a new models folder and put this in there?